backend/tests/unit/services/test_parallel_stage_executor.py

```
@@ -0,0 +1,594 @@
+        stage_manager.update_stage_execution_completed = AsyncMock()
+        
+        executor = ParallelStageExecutor(
+            agent_factory=agent_factory,
+            settings=MockFactory.create_mock_settings(),
+            stage_manager=stage_manager
+        )
+        
+        # Mock the internal _execute_parallel_stage to inspect execution_configs
+        captured_configs = []
+        
+        async def capture_configs(*args, **kwargs):
```

Ruff (0.14.8)

Unused function argument: `args`

(ARG001)